src/callbacks_step/summary.jl

```diff
@@ -14,42 +22,160 @@ function Base.show(io::IO, cb::DiscreteCallback{Condition,Affect!}) where {Condi
 end
 
 
+# Format a key/value pair for output from the SummaryCallback
+function format_key_value_line(key::AbstractString, value::AbstractString, key_width, total_width;
+                               indentation_level=0, guide='…', filler='…', prefix="│ ", suffix=" │")
+  @assert key_width < total_width
+  line  = prefix
+  if indentation_level == 0
+    squeezed_key = squeeze(key, key_width, filler=filler)
+  else
+    indentation = prefix^indentation_level
+    reduced_key_width = key_width - length(indentation)
+    squeezed_key = indentation * squeeze(key, reduced_key_width, filler=filler)
+  end
```

Comment:
```suggestion
  # Indent the key as requested (or not at all if `indentation_level == 0`)
  indentation = prefix^indentation_level
  reduced_key_width = key_width - length(indentation)
  squeezed_key = indentation * squeeze(key, reduced_key_width, filler=filler)
```
Is the `if ... else ... end` block really necessary?

Reply:
Nope. Thanks!